### PR TITLE
docs: improve docs about how to work with logs

### PR DESCRIPTION
**Pull request by georglauterbach**
# Description

1. I added information on the recently adopted `less` / `nano`
2. I added information about `/var/log/mail/`

I think users sometimes lack the information on where to find our logs and how to work with them. This PR should help clarify the issue even further.

## Type of change

- [x] Improvement (non-breaking change that does improve existing functionality)
- [x] This change is a documentation update

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code

### CODE REVIEW

File: docs/content/config/debugging.md [hunk 1]
```diff
@@ -29,22 +29,29 @@ These links may advise how the provider can unblock the port through additional
...
-2. `apt-get install <PACKAGE>`
+2. `apt-get install <PACKAGE>` to install a package, e.g., `apt-get install neovim` if you want to use NeoVim instead of `nano` (which is shipped by default).
+
+#### Logs
+
+If you need more flexibility than `docker logs` offers, within the container
+
+1. `/var/log/mail/mail.log`,
+2. `/var/log/mail/mail/<SERVICE>.log`, and
+3. `/var/log/supervisor/<SERVICE>.log`
```
Comment (polarathene) [inline docs/content/config/debugging.md:52]:
```suggestion
If you need more flexibility than what the `docker logs` command offers, then the most useful locations to get relevant DMS logs within the container are:

- `/var/log/mail/mail.log`,
- `/var/log/mail/mail/<SERVICE>.log`
- `/var/log/supervisor/<SERVICE>.log`
```

File: docs/content/config/debugging.md [hunk 2]
```diff
@@ -29,22 +29,29 @@ These links may advise how the provider can unblock the port through additional
...
 
-For example a text editor you can use in the terminal: `apt-get install nano`
+are the most useful locations to get relevant DMS logs. You may use `nano` (a text editor), `less` (a file viewer) or `tail`/`cat` to inspect (or edit) their contents.
```
Comment (polarathene) [inline docs/content/config/debugging.md:54]:
```suggestion
You may use `nano` (a text editor) to edit files, while `less` (a file viewer) and `tail`/`cat` are useful tools to inspect the contents of logs.
```

**Review by polarathene — CHANGES_REQUESTED** [review]
I'll apply the suggestions.

**Review by polarathene — DISMISSED** [review]
LGTM 👍

File: docs/content/config/debugging.md [hunk 3]
```diff
@@ -29,22 +29,29 @@ These links may advise how the provider can unblock the port through additional
...
 
 1. `apt-get update` to update repository metadata.
-2. `apt-get install <PACKAGE>`
+2. `apt-get install <PACKAGE>` to install a package, e.g., `apt-get install neovim` if you want to use NeoVim instead of `nano` (which is shipped by default).
+
+#### Logs
+
+If you need more flexibility than what the `docker logs` command offers, then the most useful locations to get relevant DMS logs within the container are:
+
+- `/var/log/mail/mail.log`,
```
Comment (polarathene) [inline docs/content/config/debugging.md:50]:
```suggestion
- `/var/log/mail/mail.log`
```

**Comment by georglauterbach** [comment]
Thanks for the review - I will merge this as you seem happy with it :D

**Pull request merged**